### PR TITLE
Update to v0.8.6

### PR DESCRIPTION
**Pull request by vinnyjth**
### How do I test this PR?

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [DEV-XXX](#url)
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android if applicable
- [ ] Set two relevant reviewers

## REVIEW

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

---

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

**Comment by IsaacHardy** [comment]
@vinnyjth Receiving some errors: 
Screen Shot 2019-05-15 at 8 35 18 AM

Already have `yarn` and `yarn nuke`.

**Comment by vinnyjth** [comment]
@IsaacHardy Can confirm. We need a few changes from Core to resolve. PR with those changes is here: https://github.com/ApollosProject/apollos-prototype/pull/715

File: packages/apollos-church-api/package.json [hunk 1]
```diff
@@ -34,17 +34,17 @@
...
-    "@apollosproject/data-connector-church-online": "^0.8.4",
-    "@apollosproject/data-connector-cloudinary": "^0.8.4",
-    "@apollosproject/data-connector-onesignal": "^0.8.4",
-    "@apollosproject/data-connector-passes": "^0.8.4",
-    "@apollosproject/data-connector-rock": "^0.8.4",
-    "@apollosproject/data-connector-twilio": "0.8.4",
-    "@apollosproject/rock-apollo-data-source": "^0.8.4",
-    "@apollosproject/server-core": "^0.8.4",
+    "@apollosproject/config": "^0.8.5",
+    "@apollosproject/data-connector-analytics": "^0.8.5",
```
Comment (IsaacHardy) [inline packages/apollos-church-api/package.json:34]: Is this supposed to be 0.8.6?

**Review by IsaacHardy — DISMISSED** [review]
LGTM

**Review by IsaacHardy — APPROVED** [review]
